scss/components/_drilldown.scss

```diff
@@ -39,7 +39,7 @@ $drilldown-background: $white !default;
     top: 0;
     #{$global-left}: 100%;
     z-index: -1;
-    height: 100%;
+    // height: 100%;
```

Comment:
Yeah, this appears to be responsible for https://github.com/zurb/foundation-sites/issues/9368.  Looking into 9239 now.